packages/botonic-plugin-ner/tsconfig.json

```diff
@@ -0,0 +1,34 @@
+{
+  "include": ["src/**/*.ts", "src/**/*.tsx"],
+  "compilerOptions": {
+    "allowUnreachableCode": false,
+    "baseUrl": "src",
+    "paths": {
+      "*": ["src/*", "dist/src/*", "types/*"]
+    },
+    "moduleResolution": "node",
+    "rootDir": "src",
+    "outDir": "dist",
```

Comment:
we use the convention in hubtype of using lib for libraries and dist for final applications like bots botonic-nlu was not following it, but now it's time to do it here. please apply it in other nlp packages (except the legacy ones) in other PRs